test/onigumo_test.exs

```diff
@@ -9,6 +9,22 @@ defmodule OnigumoTest do
 
   setup(:verify_on_exit!)
 
+  @tag :tmp_dir
+  test("run the download", %{tmp_dir: tmp_dir}) do
```

Comment:
I could not find a more suitable description for the test. Do you have any ideas?

Reply:
my suggestions to

`Onigumo.Downloader.main`:
- start
- init
- trigger

"run the download"
- activate downloading
- start download process
- integration test for downloader
- check download process

Reply:
We discussed personally, there is a option to split func `main` to `init` and `run`, where `start` should be located in `init` and rest in `main`....

Reply:
So far we keep the name `main`

Reply:
We kept `main` because the Downloader is essentialy a standalone application and the _main_ function is its entry points. The caller does not care about what the module does. `main` is a traditional name from the ages of C and Java.

Reply:
The new name of the tests is „run Downloader,“ because what we test here is that the Downloader does its job. The fact that it loadts the URLs from a file and then downloads them is a mere technical detail.